common/i2cbus.h

```diff
@@ -99,6 +110,9 @@ class I2CBus : Looper, StateMachine {
   int i;
   bool clock_detected, data_detected;
   bool i2c_detected_ = false;
+#ifdef XPOWERMAN
+  uint32_t shtimeMS = 2000;
```

Comment:
I don't understand this variable name, and it's in a very different format from everything else.
I think it should be called something ending in _time_millis_, but I don't know what, because I don't know what it does.